frontend/src/recipePage.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import './recipePage.css'
+import './recipePreview.css'
+
+export default function RecipePage({name, desc, image, ingredients, instructions}) {
+  const [newIngredient, setNewIngredient] = React.useState('');
+  const [newInstruction, setNewInstruction] = React.useState('');
```

Comment:
Since you imported `useState` on line 1, you can write `useState('')` instead of `React.useState('')`